setup.py

```diff
@@ -95,7 +95,6 @@
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
Should we add 3.10 here or do we want to do that separately?

Reply:
It requires separate effort: https://github.com/dask/distributed/issues/5350 is still an issue

Reply:
Right I guess that depends what the scope is here. In the past we have sometimes dropped and added a Python version in the same PR because of special test cases, a range of package requirements, etc. that we still wanted covered. Dropping a Python version in those cases meant we lost that test coverage, but adding a new one at the same time meant we could redistribute that workload. Not sure if that applies here yet or not. Hence the question

Reply:
> Should we add 3.10 here or do we want to do that separately?

I'd prefer to handle 3.10 support in a separate PR